src/kingsmithr1protreadmill.cpp

```diff
@@ -104,19 +105,21 @@ void kingsmithr1protreadmill::update() {
 
         QSettings settings;
         // ******************************************* virtual treadmill init *************************************
-        if (!firstInit && searchStopped && !virtualTreadMill && !virtualBike) {
+        if (!firstInit && searchStopped && !this->hasVirtualDevice()) {
```

Comment:
Note to self: find out why this class checks if searching has stopped when most others don't.

Reply:
I guess that this searchStopped is quite useless now. Since one of them is my treadmill, I can easily check it